.github/workflows/test.yml

```diff
@@ -3,10 +3,6 @@ name: Tests / Code Coverage
 #  This workflow is run on pushes to main & every Pull Requests where a .go, .mod, .sum have been changed
 on:
   workflow_call:
-  pull_request:
-  push:
-    branches:
-      - main
```

Comment:
wait why do we want to remove this? won't this get rid of running tests unless manually called?

Reply:
perhaps a better question is just where the tests are getting called now?

Reply:
they're getting called as part of the ci-release workflow:
https://github.com/celestiaorg/orchestrator-relayer/blob/17ffa8491748298ae11ff0ac77f9766d7fcbb97f/.github/workflows/ci-release.yml#L29-L34


So no need to call them twice